proxy/src/outbound.rs

```diff
@@ -54,6 +54,60 @@ impl<B> Outbound<B> {
             bind_timeout,
         }
     }
+
+    const DEFAULT_PORT: Option<u16> = Some(80);
```

Comment:
Thanks for making this a named constant! :)

Reply:
Until there's a reason not to, this should be scoped inside of `normalize`.